joomla/joomla-cms
# Invalid delete of all content articles from smart search index

### Summary of Changes

'Smart Search - Content' plugin removes all indexed articles on unpublishing any plugin with 'content' name (element).

### Testing Instructions

1. Ensure that 'Smart Search - Content' and 'Privacy - Content' plugins are enabled.
2. Ensure that you have articles indexed in Smart Search.
3. Unpublish 'Privacy - Content' plugin.

### Actual result BEFORE applying this Pull Request

All articles are removed from the search index.

### Expected result AFTER applying this Pull Request

Articles are preserved and are only removed in unpublishing of 'Smart Search - Content' plugin.

### Link to documentations
Please select:
